### DISCUSSION TITLE
Table model Json Attribute

### DISCUSSION THREAD

**Discussion by henda79** [Azure/azure-mobile-apps#762] [Q&A]
In relation to my other discussion #761 , when using Attributes to set the SQL Table name on the data model, the documents state to use either `DataTable` or Json Attributes and then links to https://www.newtonsoft.com/json/help/html/SerializationAttributes.htm

Which attribute is used ? I've used `JsonObject("MyTable")` but the resolver code does not work as it uses the `Title` property, but would work fine with the `Id` property

```
if (!string.IsNullOrWhiteSpace(jsonattr.Title))
{
    name = jsonattr.Title;
}
```
Again, is this something I am doing wrong ?

**Comment by adrianhall** [comment]
Great point - let's turn this into an issue.
